cmd/manager/main.go

```diff
@@ -59,6 +61,10 @@ func main() {
 	// controller-runtime)
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 
+	// Operator flags
+	var scanAll bool
+	pflag.BoolVar(&scanAll, "scan-all", false, "set to enable multi namespace support")
```

Comment:
Perhaps we could reuse this environment variable: https://github.com/keycloak/keycloak-operator/blob/master/deploy/operator.yaml#L24

Reply:
yes, good point. This flag is redundant, i'll remove it.